pyproject.toml

```diff
@@ -94,6 +94,7 @@ dependencies = [ # Optional
 [project.optional-dependencies] # Optional
 dev = ["check-manifest","black"]
 test = ["coverage"]
+fuzzy = ["jellyfish~=1.1"]
```

Comment:
optional dependence, do not break the exists production use

Reply:
I would make this ~1.0 to allow to float within seantic version rather tahn 1.1

Reply:
Sure, I will do it right now

Reply:
Oh, it is seems you already correct it